lntest/mock/walletcontroller.go

```diff
@@ -105,6 +105,11 @@ func (w *WalletController) ListAccounts(string,
 	return nil, nil
 }
 
+// RequiredReserve currently returns a dummy value.
+func (w *WalletController) RequiredReserve(int) btcutil.Amount {
```

Comment:
This change needs to be in the same commit where you add the method to the interface, otherwise that commit won't compile on its own.